src/gpodder/opml.py

```diff
@@ -70,10 +70,17 @@ def __init__(self, url):
         else:
             doc = xml.dom.minidom.parse(io.BytesIO(util.urlopen(url).content))
 
+        section = None
         for outline in doc.getElementsByTagName('outline'):
             # Make sure we are dealing with a valid link type (ignore case)
             otl_type = outline.getAttribute('type')
             if otl_type is None or otl_type.lower() not in self.VALID_TYPES:
+                breakpoint()
```

Comment:
Enters debugger when importing the OPML. ;)

Reply:
woopsie